addons/core/addon/styles/addon.scss

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+.filters-applied {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  margin-bottom: 1rem;
+
+  .filter-tag {
+    max-width: 200px;
+
+    .hds-tag__text {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+  }
+
+  > :not(:first-child) {
+    margin-left: 0.5rem;
+  }
```

Comment:
I think we should remove this and use a `gap: 0.5 rem` on `.filters-applied` instead since there's no vertical spacing now when the filters wrap.

<img width="1078" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/1637d0f8-13cc-4d3e-97a1-537678b955d3">

Compared to using gap: 
<img width="1098" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/7f2a4f65-a4af-458a-bb04-8ee13a02c70a">

Reply:
Good catch. Updated.